### DISCUSSION TITLE
Hide the logo in the vaults menu

### DISCUSSION THREAD

**Discussion by bywkgexmkiltxh** [Kunzisoft/KeePassDX#1954] [Ideas]
I am wondering if it's possible to permanently hide the logo in the vaults menu for aesthetic reasons. Swiping up temporarily hides the logo.
![Screenshot_20241207_204156_KeePassDX](https://github.com/user-attachments/assets/7648c38d-734d-4d52-90ae-552fa76d515f)

**Comment by J-Jamet** [comment]
It's possible, of course, but I won't hide the name of the application, which distinguishes the pro version from the normal one.
